src/overlord.rs

```diff
@@ -59,9 +58,9 @@ where
         interval: u64,
         timer_config: Option<DurationConfig>,
     ) -> ConsensusResult<()> {
-        let (mut smr_provider, evt_1, evt_2) = SMRProvider::new();
+        let (mut smr_provider, evt_1, evt_2) = SMR::new();
```

Comment:
Params's name should also be changed, such as smr_provider here.

Reply:
It is named `smr` at the next line.